sources/packages/web/src/constants/message-constants.ts

```diff
@@ -18,4 +18,5 @@ export const MINISTRY_SHORTNAME = "PSFS";
 /**
  * Ministry Fullname
  */
-export const MINISTRY_NAME = "Ministry of Post Secondary and Future Skills";
```

Comment:
This changes the Logo. Do we have the requirement to change the verbiage all around application? 
![image](https://github.com/user-attachments/assets/1ece317a-3aab-407a-9905-dd51a2acded2)

Or just about the content related to `Consent`?

Reply:
I guess that business did not anticipate it but it seems to be a desired side-effect.
I would collect some screenshots from the impacted places and add them to the ticket comments.
@JerPearson can give his approval.

Reply:
Turns out the only place that that constant is currently being used is that logo. Other places that have the ministry name include the file uploader.
<img width="1664" alt="Screenshot 2024-10-04 at 3 34 40 PM" src="https://github.com/user-attachments/assets/0bc315fe-636c-421c-b744-a8d96fc17d8a">
<img width="1624" alt="Screenshot 2024-10-04 at 3 36 21 PM" src="https://github.com/user-attachments/assets/da0df13b-a1d8-4b54-8367-c401600b4bb7">
<img width="330" alt="Screenshot 2024-10-04 at 3 41 57 PM" src="https://github.com/user-attachments/assets/e77ae200-395c-4fcd-a173-2752ffaf1f3a">

Reply:
Jeremy's away today, Josh took a look and approved the additional change. :)